Gemfile

```diff
@@ -4,6 +4,9 @@ source "https://rubygems.org"
 
 ruby "2.7.2"
 
+# To load environment variables from .env into ENV
+gem "dotenv-rails"
```

Comment:
@unnitallman Do we need this in all environments?

Reply:
`development` and `test` will do usually since Heroku has its own setup for loading ENV variables. But this PR is related to running `staging` env on local - which means we need a way to load ENV variables from `.env` file on local machine as well. Hence added the gem without development/test scope.

Reply:
@unnitallman I see just the gem being added in the PR. Can you add instruction to run `staging` env on local or link to that.

Reply:
That is done. That was the purpose of the issue for which I have sent this PR: https://github.com/bigbinary/neeto-help-web/issues/7857. Please see the issue description and the first comment.